packages/bpmn-editor/src/diagram/nodes/NodeStyle.ts

```diff
@@ -106,7 +106,7 @@ export function getNodeLabelPosition({ nodeType }: { nodeType: BpmnNodeType }):
     case NODE_TYPES.group:
     case NODE_TYPES.textAnnotation:
     case NODE_TYPES.lane:
-      return "top-left";
+      return "center-left-vertical";
```

Comment:
Group and Text Annotation nodes shouldn't have their labels in vertical position...

Reply:
after such change, is still `"top-left"` used somewhere or we can remove it completely?